resources/fileTemplates/internal/Magento Eav Attribute Data Patch Class.php.ft

```diff
@@ -58,6 +58,7 @@ class ${CLASS_NAME} implements ${IMPLEMENTS} {
             #end
 ]
         );
+        return $this;
```

Comment:
But this is completely useless. See `vendor/magento/framework/Setup/Patch/PatchApplier.php:157`. I think the issue related rather to the wrong doc block.
Moreover, I see no patches in the core implemented in such a way.

Reply:
Hi @VitaliyBoyko, totally agree with you 💯. But checking the `\Magento\Framework\Setup\Patch\PatchInterface::apply`, we may see the `return` annotation.

> Moreover, I see no patches in the core implemented in such a way.

Unfortunately, there are many places like that:
- \Magento\Analytics\Setup\Patch\Data\ActivateDataCollection::apply
- \Magento\Analytics\Setup\Patch\Data\PrepareInitialConfig::apply
- \Magento\Customer\Setup\Patch\Data\MigrateStoresAllowedCountriesToWebsite::apply

But I agree that there is no reason of it.

Reply:
I think that annotation is wrong. 😑 We do not need to support it in the plugin

Reply:
Yeap 👍, removed it